### PR TITLE
bump nodejs and remove freebsd keywords to fix CI

### PR DESCRIPTION
**Pull request by dg98**
I dropped various nodejs versions that aren't present in the main tree anymore. I also noticed that all versions are in the testing branches, is it intentional that none were moved to stable like in the main tree?
I also removed the freebsd keywords because all freebsd packages are being removed due to [#683284](https://bugs.gentoo.org/683284#c10) and they were causing the CI to fail.

### CODE REVIEW

**Comment by blueness** [comment]
pushed

File: net-libs/nodejs/nodejs-12.12.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,212 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="threads"
+inherit bash-completion-r1 flag-o-matic pax-utils python-any-r1 toolchain-funcs
```
Comment (steils) [inline net-libs/nodejs/nodejs-12.12.0.ebuild]: inherit bash-completion-r1 flag-o-matic pax-utils python-any-r1 toolchain-funcs xdg-utils

File: net-libs/nodejs/nodejs-12.12.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+	# debug builds. change install path, remove optimisations and override buildtype
+	if use debug; then
+		sed -i -e "s|out/Release/|out/Debug/|g" tools/install.py || die
+		BUILDTYPE=Debug
+	fi
+
+	default
+}
+
+src_configure() {
```
Comment (steils) [inline net-libs/nodejs/nodejs-12.12.0.ebuild]: xdg_environment_reset
